modules/vaos/app/services/common/jwt_wrapper.rb

```diff
@@ -19,22 +19,14 @@ def initialize(service_settings, service_config)
     end
 
     def sign_assertion
-      JWT.encode(claims, rsa_key, SIGNING_ALGORITHM, jwt_headers)
+      @rsa_key ||= OpenSSL::PKey::RSA.new(File.read(key_path))
+      JWT.encode(claims, @rsa_key, SIGNING_ALGORITHM, jwt_headers)
     rescue ConfigurationError => e
       Rails.logger.error("Service Configuration Error: #{e.message}")
       raise VAOS::Exceptions::ConfigurationError.new(e, @config.service_name)
-    end
-
-    def rsa_key
-      raise ConfigurationError, 'RSA key path is not configured' if key_path.blank?
-
-      raise ConfigurationError, "RSA key file not found at: #{key_path}" unless File.exist?(key_path)
-
-      @rsa_key ||= begin
-        OpenSSL::PKey::RSA.new(File.read(key_path))
-      rescue => e
-        raise ConfigurationError, "Failed to load RSA key: #{e.message}"
-      end
+    rescue Errno::ENOENT => e
+      Rails.logger.error("Key file not found: #{e.message}")
+      raise VAOS::Exceptions::ConfigurationError.new(e, @config.service_name)
```

Comment:
You should put all the original code back, otherwise we will get weird exception output in the API responses. Just revert to the old code.